Mayhemfile

```diff
@@ -0,0 +1,6 @@
+project: rnshah9/zapcc
+target: zapcc
+
+cmds:
+  - cmd: /zapcc/build/bin/zapcc --config @@ /zapcc/build/bin/test.c
```

Comment:
It looks like this may be the incorrect usage of the `filepath` directive again. Please address.